### PR TITLE
Validate VK_EXT_robustness2

### PR DESCRIPTION
**Pull request by jeffbolznv**

### CODE REVIEW

File: layers/parameter_validation_utils.cpp [hunk 1]
```diff
@@ -2591,30 +2598,15 @@ bool StatelessValidation::validate_WriteDescriptorSet(const char *vkCallingFunct
...
-                    // If descriptorType is VK_DESCRIPTOR_TYPE_COMBINED_IMAGE_SAMPLER, VK_DESCRIPTOR_TYPE_SAMPLED_IMAGE,
-                    // VK_DESCRIPTOR_TYPE_STORAGE_IMAGE or VK_DESCRIPTOR_TYPE_INPUT_ATTACHMENT, the imageView and imageLayout
-                    // members of any given element of pImageInfo must be a valid VkImageView and VkImageLayout, respectively
-                    for (uint32_t descriptor_index = 0; descriptor_index < pDescriptorWrites[i].descriptorCount;
-                         ++descriptor_index) {
-                        skip |= validate_required_handle(vkCallingFunction,
-                                                         ParameterName("pDescriptorWrites[%i].pImageInfo[%i].imageView",
-                                                                       ParameterName::IndexVector{i, descriptor_index}),
-                                                         pDescriptorWrites[i].pImageInfo[descriptor_index].imageView);
-                        skip |= validate_ranged_enum(vkCallingFunction,
```
Comment (Tony-LunarG) [inline layers/parameter_validation_utils.cpp:2612]: Was this removal intentional?  I don't see this validated anywhere else on this branch.

Comment (jeffbolznv) [reply layers/parameter_validation_utils.cpp:2612]: CoreChecks::ValidateImageUpdate has some code that checks descriptor type against imageLayout, but it may not be complete. I could restore the imageLayout checks here if you want.

Comment (Tony-LunarG) [reply layers/parameter_validation_utils.cpp:2612]: If you don't mind.  The error message I get from ValidateImageUpdate when I use a bad enum isn't as clear as this error message.  Thanks!

Comment (jeffbolznv) [reply layers/parameter_validation_utils.cpp:2612]: OK, I've added back the imagelayout check.
